docs/docs/loops-conditional-rendering.md

```diff
@@ -0,0 +1,228 @@
+---
+id: forms
+title: Loops & Conditional Rendering
+permalink: docs/loops-conditional-rendering.html
+prev: 
+next: 
+---
+
+
+## Lists
+
+### A Basic List
+
+To render an array or collection of objects into a list use the ES6 `map()`.
+
+```javascript
+render() {
+  let persons = ['Ben', 'Chris', 'Dan', 'Paul', 'Tom'];
+  return (
+    <div>
+      <ul>
+        {persons.map((person) => {
+          return (
+            <li>{person}</li>
+          );
+        })}
+      </ul>
+    </div>
+  );
+}
+```
+> Using a fat arrow function (`() => {}`) will automatically bind `this` to the function.
+
+### A Basic List Using Keyed Components
+
+In this example we loop through a collection of objects, then pass props to a `Person` component. In our `map()` function we provide a second argument `index` that provides us with the current index of our loop. We use this index as our key, however more complex data benefits from a unique identifier (like a database object id).
```

Comment:
It is slightly misleading to say that only complex data would benefit from unique identifiers as keys. If the order of the items could possibly change in any way, using the index as the `key` would be a bad idea.